qiskit_experiments/library/tomography/qpt_experiment.py

```diff
@@ -91,6 +93,13 @@ def __init__(
                 preparation basis index, and ``m[i]`` is the measurement basis index
                 for qubit-i. If not specified full tomography for all indices of the
                 preparation and measurement bases will be performed.
+            conditional_measurement_indices: Optional, a subset of `measurement_indices`
```

Comment:
> What happens if you specify circuit with conditional operation without specifying this option?

This comment was because I misunderstood _conditional_ operation.

Reply:
This is to specify which clbits in the circuit you are performing tomography on to treat as conditional. If you dont specify any, its the same as current tomography where they are all marginalized over.

I didn't really see the value of adding another experiment subclass to do this when it would be the same as existing QPT/QST but with these two kwargs added.

Reply:
See my comment further below, but this refers to how to treat tomographic measurements added to a circuit as conditional or standard in the tomographic reconstruction. It doesnt matter if the circuit already contains measurements/conditional gates or not.

Reply:
I could remove the conditional_measurement_indices from QST and QPT for now (but leave them in base TomographyExperiment for future use in a separate subclass), and only leave the conditional_circuit_clbits one which is necessary for handling tomograpahy of a specific circuit containing measurements/clbits.

Reply:
I read the rest of your comments. I feel we just need more clear definition of conditional, i.e. it is not for the conditional gate, but tomographic reconstruction conditioned on the measured subsystem.